lib/litle.rb

```diff
@@ -94,6 +97,15 @@ def parse(kind, xml)
 
         parsed
       end
+
+      def success_from(kind, parsed)
+        if kind == :registerToken
+          %w(000 801 802).include?(parsed[:response])
+        else
+          %w(000 010).include?(parsed[:response])
```

Comment:
It looks like the main difference with upstream is the support of `010`? Could you document it?